world/concert_with_human.sdf

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" ?>
+<sdf version="1.4">
+
+    <world name="default">
+
+        <!-- simulation time support  -->
+        <plugin name="xbot2_clock" filename="libxbot2_gz_clock.so"/>
+
+        <gravity>0 0 -9.81</gravity>
+        
+        <include>
+            <file>/home/user/tum_integration_ws/catkin_ws/src/human-gazebo/models/human_work</file>
+            <!-- TODO: https://answers.gazebosim.org//question/16159/how-to-include-uri-relatively/-->
+        </include>
+
+
+        <model name="ground_plane">
+            <static>true</static>
+            <link name="link">
+                <collision name="collision">
+                    <geometry>
+                        <plane>
+                            <normal>0 0 1</normal>
+                            <size>100 100</size>
+                        </plane>
+                    </geometry>
+                    <surface>
+                        <friction>
+                            <ode>
+                                <mu>1</mu>
+                                <mu2>1</mu2>
+                            </ode>
+                        </friction>
+                    </surface>
+                </collision>
+                <visual name="visual">
+                    <cast_shadows>false</cast_shadows>
+                    <geometry>
+                        <plane>
+                            <normal>0 0 1</normal>
+                            <size>100 100</size>
+                        </plane>
+                    </geometry>
+                    <material>
+                        <script>
+                            <uri>file://media/materials/scripts/gazebo.material</uri>
+                            <name>Gazebo/Grey</name>
+                        </script>
+                    </material>
+                </visual>
+            </link>
+        </model>
+
+        <!-- Global light source -->
+        <include>
+            <uri>model://sun</uri>
+        </include>
+
+        <!-- Focus camera on tall pendulum -->
+        <gui fullscreen='0'>
+            <camera name='user_camera'>
+                <pose>4.927360 -4.376610 3.740080 0.000000 0.275643 2.356190</pose>
+                <view_controller>orbit</view_controller>
+            </camera>
+        </gui>
+
+        <physics name="update_rate" type="ode">
+            <real_time_update_rate>2000</real_time_update_rate>
```

Comment:
This is quite fast. Do we need 2000? 1k or even 500 could maybe be enough?

Reply:
This was the default update rate with the normal concert_descriptions `empty_world.sdf` . I would leave it like that, since I think that problems can occur, when the physics rate is too slow (too high velocities, clipping through planes and stuff like that)